src/components/FileTree/presenter.jsx

```diff
@@ -52,7 +55,7 @@ class FileTree extends Component {
     this.getNode = this.getNode.bind(this);
   }
 
-  getNodeToggler({ classes, docType, isLoading, isExpanded }) {
+  getNodeToggler({ classes, docType, isLoading, isExpanded, isHome }) {
```

Comment:
I do not like "isHome" param, which is too specific and needed only for one node